R/initialize.R

```diff
@@ -82,6 +84,10 @@ initialize_indent <- function(pd_flat) {
   pd_flat
 }
 
+
+
+
+
```

Comment:
What's the semantic difference between 1 empty line, 3 empty lines (as in line 22) and 5 empty lines?